lib/Profile.dart

```diff
@@ -0,0 +1,227 @@
+import 'dart:io';
+import 'dart:ui';
+
+import 'package:flutter/material.dart';
+import 'package:flutter/rendering.dart';
+import 'package:image_picker/image_picker.dart';
+
+class Profile extends StatefulWidget {
+  Profile({Key key}) : super(key: key);
+  @override
+  _ProfileState createState() => _ProfileState();
+}
+
+class _ProfileState extends State<Profile> {
+  File _image;
+
+  Future getImage() async {
+    var image = await ImagePicker.pickImage(source: ImageSource.gallery);
+    setState(() {
+      _image = image;
+    });
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    double _width = MediaQuery.of(context).size.width;
+    double _picDiameter = _width * 0.27;
+    double _picRadius = _picDiameter / 2;
+    double _picMarginLR = _picDiameter / 6.25;
+    double _picMarginTB = _picDiameter / 4;
+    double _picBtnDiameter = _picDiameter * 0.39;
+
+    return Column(
+        crossAxisAlignment: CrossAxisAlignment.start,
+        children: [
+          Padding(
+            padding: EdgeInsets.only(
+                left: 24.0,
+                top: 18.0 + MediaQuery.of(context).padding.top,
+                bottom: 16.0
+            ),
+            child: Text(
+                'Your Profile',
+                style: Theme.of(context).textTheme.headline
+            ),
+          ),
+          Container(
+              decoration: BoxDecoration(
+                color: Colors.white,
+                borderRadius: BorderRadius.circular(3),
+                boxShadow: [
+                  BoxShadow(
+                      color: Color.fromARGB(15, 0, 0, 0),
+                      offset: Offset(0, 4.0),
+                      blurRadius: 10.0,
+                      spreadRadius: 1.0
+                  )
+                ],
+              ),
+              child: Row(
+                  children: [
+                    Padding (
+                        padding: EdgeInsets.only(left: _picMarginLR, right: _picMarginLR, top: _picMarginTB, bottom: _picMarginTB),
+                        child: Stack (
+                          children: [
+                            Padding (
+                                padding: EdgeInsets.only(bottom: _picDiameter * 0.05),
+                                child: CircleAvatar(
+                                  backgroundImage: _image == null
+                                      ? AssetImage('assets/images/terry.jpg')
+                                      : FileImage(_image),
```

Comment:
As mentioned earlier, this will eventually be the image fetched from the backend.